sections/futures/TradeCrossMargin/CrossMarginFeesBox.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+import InfoBox from 'components/InfoBox';
+import { useFuturesContext } from 'contexts/FuturesContext';
+import { formatDollars } from 'utils/formatters/number';
+
+export default function CrossMarginFeesBox() {
+	const { tradeFees } = useFuturesContext();
+
+	return (
+		<StyledInfoBox
```

Comment:
Can we also display the fee percentage here? Or can we take note to add the fee breakdown?

Reply:
Added a breakdown of the fees